### PR TITLE
Add a script to deploy to a cluster

### PR DESCRIPTION
**Pull request by brianjmurrell**
A script to make it easy to deploy a build to a cluster
including a dev cluster or a Vagrant cluster, etc.

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

---
This change is [Reviewable](https://reviewable.io/reviews/intel-hpdd/intel-manager-for-lustre/428)

### CODE REVIEW

File: scripts/deploy_build [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ALL_NODES="${HOST}vm${CLUSTER}[2-9]"
+EE_VERSION=$(make -s -f include/Makefile.version .ieel_version)
+# if you want a release from jenkins:
+#EE_JOB_NUM=292
+# otherwise:
+ARCHIVE_NAME="iml-$EE_VERSION.tar.gz"
+
+CHROMA_DIR=.
+CHROMA_USER="admin"
+CHROMA_EMAIL="brian.murrell@intel.com"
```
Comment (jgrund) [inline scripts/deploy_build]: Can we make this a more generic email?

Comment (brianjmurrell) [reply scripts/deploy_build]: Sure.  Any suggestions of what we have that is more generic?

Comment (brianjmurrell) [reply scripts/deploy_build]: Any suggestions of what to use here @jgrund?

Comment (jgrund) [reply scripts/deploy_build]: sorry, just saw this:

iml@intel.com

Comment (brianjmurrell) [reply scripts/deploy_build]: iml@intel.com is a valid e-mail address?

Comment (jgrund) [reply scripts/deploy_build]: yes.

File: scripts/deploy_build [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+# if this node uses the Intel proxies, make sure the agent is doing so
+if [ -f /etc/profile.d/intel_proxy.sh ]; then
+    echo \". /etc/profile.d/intel_proxy.sh\" > /etc/sysconfig/chroma-agent
+fi
+
+yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/repo/epel-7/managerforlustre-manager-for-lustre-epel-7.repo
+yum-config-manager --add-repo http://mirror.centos.org/centos/7/extras/x86_64/
+yum -y install epel-release
+#yum-config-manager --add-repo https://build.whamcloud.com/job/lustre-master/lastSuccessfulBuild/arch=x86_64,build_type=server,distro=el7,ib_stack=inkernel/artifact/artifacts/
```
Comment (tanabarr) [inline scripts/deploy_build]: commented code, should be removed or documented

**Comment by brianjmurrell** [comment]
@tanabarr As the comment says, you would use those when you want to test lustre-master.

**Pull request merged**